bulldozer/signals.go

```diff
@@ -140,6 +148,7 @@ func (s Signals) MatchesAny(ctx context.Context, pullCtx pull.Context, tag strin
 		&s.PRBodySubstrings,
 		&s.Branches,
 		&s.BranchPatterns,
```

Comment:
I noticed that `MaxCommits` signal is missing here. Is this intentional?

Reply:
Good catch, but I believe this is intentional for now. When we added the `MaxCommits` signal, it was specifically for the `mege_method` block, which is the only block that uses `MatchesAll` right now. We may want to change that in the future, but I think we can leave it alone for now.